app/lib/aws/package.scala

```diff
@@ -8,7 +8,6 @@ package object aws {
 
   lazy val CredentialsProvider = new AWSCredentialsProviderChain(
     new ProfileCredentialsProvider(ProfileName),
-    new InstanceProfileCredentialsProvider(false),
-    new EnvironmentVariableCredentialsProvider()
+    new InstanceProfileCredentialsProvider(false)
```

Comment:
I know it's not related to this change but is it possible to explicitly name the parameter here? What is it that's `false`?